ingestion_server/test/generate_integration_test_docker_compose.py

```diff
@@ -55,6 +56,9 @@
         ingestion_server['depends_on'] = ['integration-es', 'integration-db']
         ingestion_server['build'] = '../'
 
+        # Set the directory correctly in the volume
+        ingestion_server['volumes'] = ['../:/ingestion_server']
+
```

Comment:
Seems like this change was needed because of #137.

Reply:
Yes, but it's much better how it is now :)